.github/workflows/njsscan-analysis.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This workflow integrates njsscan with GitHub's Code Scanning feature
+# nodejsscan is a static security code scanner that finds insecure code patterns in your Node.js applications
+
+name: njsscan sarif
```

Comment:
Doesn't this do the same thing as CodeQL?

Reply:
I just threw the whole kitchen sink here ^^

I can remove it if you don't want this scanner.